nixos/modules/services/misc/paperless.nix

```
@@ -252,9 +252,59 @@ in
         '';
       };
     };
+
+    exporter = {
+
+      enable = lib.mkEnableOption "regular automatic document exports.";
+
+      directory = lib.mkOption {
+        type = lib.types.str;
+        default = cfg.dataDir + "/exports";
```

nit: This will eval with the default value of dataDir inserted. Perhaps add an explicit placeholder to make it clear to users that this depends on dataDir. 

```suggestion
        default = cfg.dataDir + "/exports";
        defaultText = "\${dataDir}/exports";
```

---

I just saw a CI check failed likely because of this. I didn't know we were enforcing this but that's probably a good thing.

This must happen then.